.storybook/webpack.config.js

```diff
@@ -2,17 +2,17 @@ const path = require('path')
 const genDefaultConfig = require('@storybook/react/dist/server/config/defaults/webpack.config.js')
 
 module.exports = (config, env) => {
-    const myConfig = genDefaultConfig(config, env)
+  const myConfig = genDefaultConfig(config, env)
```

Comment:
hah...sorry... my editors default is 4 spaces because work requires 4 spaces. I will try and keep this from happening again.